examples/lazytensor_tanh.py

```diff
@@ -26,9 +26,9 @@
     import RefBackendLinalgOnTensorsBackend
 from torch_mlir.passmanager import PassManager
 
-from utils.annotator import Annotation
-from utils.torch_mlir_types import TorchTensorType
-from lazytensor.builder import build_module
+from torch_mlir.eager_mode.annotator import Annotation
```

Comment:
Can you please copy this code and have it be an internal detail of eager_mode instead of moving it? I really don't think this is a public API that we want people to be seeing. Most of these examples are out of date anyway.

Also, can you add a TODO on Annotation that this is really something that is coupled to and belongs next to `ModuleBuilder.import_function`